apps/src/templates/teacherDashboard/teacherSectionsRedux.js

```diff
@@ -875,14 +873,13 @@ export default function teacherSections(state = initialState, action) {
     }
 
     const lessonExtraSettings = {};
+    if (action.props.scriptId && !action.props.lessonExtras) {
+      lessonExtraSettings.lessonExtras = true;
```

Comment:
Nit: for someone new coming in, it may not be clear why this is being set to true. I think keeping the variable `defaultLessonExtras` makes the intent more clear